package.json

```diff
@@ -64,18 +66,17 @@
         "dom-align": "1.12.3",
         "lottie-web": "5.8.1",
         "memoize-one": "6.0.0",
+        "react": "17.0.2",
+        "react-dom": "17.0.2",
         "react-flip-toolkit": "7.0.13",
         "react-is": "18.1.0",
         "resize-observer-polyfill": "1.5.1",
         "scroll-into-view-if-needed": "2.2.25",
         "shallowequal": "1.1.0"
     },
     "peerDependencies": {
-        "@types/react": "17.0.39",
-        "@types/react-dom": "17.0.11",
-        "react": "17.0.2",
-        "react-app-polyfill": "3.0.0",
-        "react-dom": "17.0.2"
+        "react": ">=16.8",
```

Comment:
Why would this be necessary if React is in deps? To trigger a warning at compile time in the host app? Moving React into deps may cause module collisions in the host. @ychhabra-eightfold - would like your take on this too.

Reply:
It would really be about the warnings, but I think following the model ant uses is probably a better approach.